src/components/Board.js

```diff
@@ -4,32 +4,37 @@ import Square from './Square';
 import PropTypes from 'prop-types';
 
 
-const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
-  // squares is a 2D Array, but 
-  //  you need to return a 1D array
-  //  of square components
+const Board = ({ squares, onClickCallback, checkWinner }) => {
+  const generateSquareComponents = (squares, onClickCallback) => {
+    const singleArraySquares = [].concat(...squares);
+    return singleArraySquares.map((square) => {
+      return (
+        <Square
+          value={square.value}
+          id={square.id}
+          onClickCallback={onClickCallback}
+          key={square.id}
+        />
+      );
+    });
+  };
 
-}
-
-const Board = ({ squares, onClickCallback }) => {
   const squareList = generateSquareComponents(squares, onClickCallback);
   console.log(squareList);
```

Comment:
I think we left this for debugging purposes for you, but let's always get rid of any print statements before final submission

```suggestion

```